CHANGELOG.md

```diff
@@ -1,21 +1,15 @@
 # CERTIFICATE-TRANSPARENCY-GO Changelog
 
-## HEAD
+## v1.1.4
 
-### Integration
-
- * Breaking change to API for `integration.HammerCTLog`:
-    * Added `ctx` as first argument, and terminate loop if it becomes cancelled
+[Published 2022-10-21](https://github.com/google/certificate-transparency-go/releases/tag/v1.1.4)
 
 ### JSONClient
 
  * PostAndParseWithRetry now does backoff-and-retry upon receiving HTTP 429.
```

Comment:
Wasn't this in 1.1.3 according to https://github.com/google/certificate-transparency-go/releases?

Reply:
Good catch. I have moved it to 1.1.3.